maps/southern_cross/southern_cross_defines.dm

```diff
@@ -153,6 +153,7 @@
 	flags = MAP_LEVEL_STATION|MAP_LEVEL_CONTACT|MAP_LEVEL_PLAYER|MAP_LEVEL_CONSOLES
 	holomap_legend_x = 220
 	holomap_legend_y = 160
+	event_regions = list(EVENT_REGION_SPACESTATION, EVENT_REGION_PLAYER_MAIN_AREA)
```

Comment:
Note: y'all will need to either disable this system, or copy these lists into your maps defines.

Reply:
We don't really use gamemaster either way, we still use old event containers.